code/modules/events/minor/special_order.dm

```diff
@@ -0,0 +1,401 @@
+
+		New()
+			if(!breakfast)
+				breakfast = list()
+				for(var/food_type in concrete_typesof(/obj/item/reagent_containers/food/snacks))
+					var/obj/item/reagent_containers/food/snacks/F = food_type
+					if(initial(F.meal_time_flags) & MEAL_TIME_BREAKFAST)
+						breakfast += food_type
+			for(var/i in 1 to rand(3,6))
+				src.food_order[pick(breakfast)] = pick(60; 1, 30; 2, 10; 3)
+			..()
+
+	lunch_order
+		name = "Lunch Order"
+
+		New()
+			if(!lunch)
+				lunch = list()
+				for(var/food_type in concrete_typesof(/obj/item/reagent_containers/food/snacks))
+					var/obj/item/reagent_containers/food/snacks/F = food_type
+					if(initial(F.meal_time_flags) & MEAL_TIME_LUNCH)
+						lunch += food_type
+			for(var/i in 1 to rand(3,6))
+				src.food_order[pick(lunch)] = pick(60; 1, 30; 2)
+			..()
+
+	dinner_order
+		name = "Dinner Order"
+
+		New()
+			if(!dinner)
+				dinner = list()
+				for(var/food_type in concrete_typesof(/obj/item/reagent_containers/food/snacks))
+					var/obj/item/reagent_containers/food/snacks/F = food_type
+					if(initial(F.meal_time_flags) & MEAL_TIME_DINNER)
+						dinner += food_type
+			for(var/i in 1 to rand(3,6))
+				src.food_order[pick(dinner)] = pick(60; 1, 30; 2)
+			..()
+
+	snack_order
+		name = "Snack Order"
+
+		New()
+			if(!snacks)
+				snacks = list()
+				for(var/food_type in concrete_typesof(/obj/item/reagent_containers/food/snacks))
+					var/obj/item/reagent_containers/food/snacks/F = food_type
+					if(initial(F.meal_time_flags) & MEAL_TIME_SNACK)
+						snacks += food_type
+			for(var/i in 1 to rand(3,6))
+				src.food_order[pick(snacks)] = pick(40; 1, 40; 2, 20; 3)
+			..()
+
+/datum/special_order/reagents/blood
+	name = "Blood Request"
+	requisition = new /obj/item/paper/requisition/blood
+	price = 9000
+	reagents_list = list("blood"=1000)
+
+/datum/special_order/surgery/organ_swap
+	name = "Organ Swap"
+	weight = 25
+	price = 5000
+	C = new /obj/storage/crate/wooden
+	requisition = new /obj/item/paper/requisition/surgery/organ_swap
+	var/mob/living/carbon/human/target
+	var/target_organs = list()
+
+	New()
+		..()
+		var/possible_targets = list("brain", "left_eye", "right_eye", "heart", "left_lung", "right_lung", "butt", "left_kidney", "right_kidney", "liver", "stomach", "intestines", "spleen", "pancreas", "appendix")
+		for(var/i in 1 to rand(3,6))
+			target_organs |= pick(possible_targets)
+
+	pack_crate()
+		//Make Mob
+		target = new /mob/living/carbon/human/npc/assistant
+		randomize_look(target, 1, 1, 1, 1, 1, 0)
+		target.set_loc(C)
+		target.TakeDamage("All", rand(10, 20), rand(10, 20))
+		target.organHolder.damage_organs(1, 6, 10, target_organs)
+		//Let people have time to figure out what is going on before he starts fucking shit up
+		target.setStatus("paralysis", rand(4 MINUTE, 6 MINUTES))
+		target.setStatus("stunned", rand(4 MINUTE, 4 MINUTES))
+		target.setStatus("weakened", rand(1 MINUTE, 2 MINUTES))
+		//Fuck up Organs
+		target.sleeping = 10 MINUTES
```

Comment:
statuses are capped at 30 seconds, `sleeping` is measured in (mult-adjusted) life loops (approx 20 DS), not deciseconds

Reply:
That explains some things...

Reply:
Removed and drugged